renku/cli/config.py

```diff
@@ -46,13 +53,19 @@
 @click.command()
 @click.argument('key', required=True)
 @click.argument('value', required=False, default=None)
+@click.option(
+    '--local',
```

Comment:
If local is the default, does it need a flag? Just to be explicit/more readable?

Reply:
local is the default for storing. When reading, local and global are both read and keys in local override similar keys in global. `--local` in used only to limit the read to local config.

This is something I copied from Git. If this is confusing I can remove it.

Reply:
No it's fine, I realized what the point was after I started playing with it. But it's only useful if we also show the configuration, e.g. `renku config` without arguments would show you the combined configurations and `renku config --local` only the options you have set for the current project.

Reply:
Done.